### PR TITLE
fix: do not show session expired modal while routing to sign-in page

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

- Fixed bug that shows session expired modal again while routing to sign-in page.
- Refactored App.vue to use script setup.

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!!

**Review by piggggggggy — APPROVED** [review]
LGTM!

File: apps/web/src/App.vue [hunk 1]
```diff
@@ -1,24 +1,91 @@
...
+
+import MobileGuideModal from '@/services/auth/password/MobileGuideModal.vue';
+import { AUTH_ROUTE } from '@/services/auth/route-config';
+
+const vm = getCurrentInstance()?.proxy as Vue;
+
+const state = reactive({
+    showGNB: computed(() => vm.$route.matched[0]?.name === 'root'),
+    isExpired: computed(() => !state.isRoutingToSignIn && store.state.error.visibleSessionExpiredError && getRouteAccessLevel(vm.$route) >= ACCESS_LEVEL.AUTHENTICATED),
+    isRoutingToSignIn: false,
```
Comment (sulmoJ) [inline apps/web/src/App.vue:37]: 🔫

**Review by sulmoJ — APPROVED** [review]
LGTM~

**Pull request merged**